docs/en/query_language/agg_functions/reference.md

```diff
@@ -1,11 +1,65 @@
-# Function reference
+# Function Reference
 
-## count() {#agg_function-count}
+## count {#agg_function-count}
 
-Counts the number of rows. Accepts zero arguments and returns UInt64.
-The syntax `COUNT(DISTINCT x)` is not supported. The separate `uniq` aggregate function exists for this purpose.
+Counts the number of rows.
+
+```
+count(expr)
+```
+
+**Parameters**
+
+The function can take zero or one parameter.
+
+- If the function is called without parameters it counts the number of rows. The `count()` syntax is a ClickHouse-specific implementation. The most other DBMS suggest `COUNT(*)` usage.
+- If the [expression](../syntax.md#syntax-expressions) is passed, then the function counts how many times this expression returned not null. If the expression returns a value of the [Nullable](../../data_types/nullable.md) data type, then the result of `count` stays not `Nullable`. The function returns 0 if all the expressions returned `NULL`.
```

Comment:
* it looks like most of this description belongs to "Returned value" section
* "all the expressions" sounds incorrect, there's only one expression